src/SwedbankPay.Sdk.Tests/TestBuilders/PaymentRequestBuilder.cs

```diff
@@ -174,43 +179,54 @@
         public PaymentRequestBuilder WithInvoiceTestValues(Guid payeeId, Operation operation = null)
         {
             this.operation = operation ?? Operation.FinancingConsumer;
-            this.intent = Intent.Authorization;
+            this.intent = PaymentIntent.Authorization;
             this.currency = new CurrencyCode("NOK");
             this.description = "Test Description";
             this.payerReference = "AB1234";
             this.userAgent = "useragent";
             this.language = new CultureInfo("nb-NO");
-            this.urls = new Urls(new List<Uri> { new Uri("https://example.com") }, new Uri("https://example.com/payment-completed"), new Uri("https://example.com/termsandconditoons.pdf"), new Uri("https://example.com/payment-canceled"));
+            this.urls = new Urls(new UrlsDto
+            {
+                HostUrls = new List<Uri> { new Uri("https://example.com") },
+                CallbackUrl = new Uri("https://example.com/payment-completed"),
+                TermsOfServiceUrl = new Uri("https://example.com/termsandconditoons.pdf"),
+                CancelUrl = new Uri("https://example.com/payment-canceled")
+            });
             this.payeeInfo = new PayeeInfo(payeeId, DateTime.Now.Ticks.ToString());
             this.generatePaymentToken = false;
-            this.amount = Amount.FromDecimal(1600);
-            this.vatAmount = Amount.FromDecimal(0);
-            this.metadata = new Dictionary<string, object> { { "key1", "value1" }, { "key2", 2 }, { "key3", 3.1 }, { "key4", false } };
+            this.amount = new Amount(1600);
+            this.vatAmount = new Amount(0);
+            this.metadata = new MetadataResponse { { "key1", "value1" }, { "key2", 2 }, { "key3", 3.1 }, { "key4", false } };
             this.invoiceType = InvoiceType.PayExFinancingNO;
-            this.price = new List<Price>
+            this.price = new List<IPrice>
             {
                 new Price(this.amount, PriceType.Invoice, this.vatAmount)
             };
             return this;
         }
 
-        public PaymentRequestBuilder WithVippsTestValues(Guid payeeId, Operation operation = null, Intent intent = Intent.Authorization)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)